CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 1.11.2 - 2024-09-23
+
+### Fixed
+
+- Fix numerous debugging issues on Windows ([#1083](https://github.com/swiftlang/vscode-swift/pull/1083))
```

Comment:
I would use the same wording as the PR here `Fixed several debugging issues on Windows`

Reply:
done